src/sentry/static/sentry/app/components/activity/item/avatar.jsx

```diff
@@ -50,3 +50,11 @@ const SystemAvatar = styled('span')`
 const Logo = styled(InlineSvg)`
   color: ${p => p.theme.gray5};
 `;
+
+const Placeholder = styled('div')`
+  height: ${p => p.size}px;
+  width: ${p => p.size}px;
+  border-radius: 100%;
+  background-color: #f5f5f5;
```

Comment:
Can we use one of the them colours?

Reply:
Gonna keep it as is, the bg is == `whiteDark`, `offWhite` is too light, and `offWhite2` is too purple/dark. We'll do a design pass once everything else is in.